### DISCUSSION TITLE
Namecheap disabled my domain for phising/fraud

### DISCUSSION THREAD

**Discussion by ilbrigz** [caprover/caprover#1823] [General]
I just found out about caprover. I deployed it  and pointed it to a namecheap A Record. I enjoyed the experience of deploying services in just few clicks. I ended up deploying and redeploying a lot of your amazing built in services many of which I just heard of. I was the only one accessing the site and was using 3 of my devices and maybe a VPN once.

Somehow after 3 days of deploying and redeploying services, namecheap flagged and suspended my account for fraud/phising. I contacted their live support and he failed to explain to me why such thing happened.

What can I do to prevent this in the future? Where do you think should I buy my domain name next?

**Comment by githubsaturn** [comment]
Thanks for the kind words! 

The domain registration is outside of CapRover scope, but there is a number of good registrars out there: Google Domains, GoDaddy, Cloudflare etc

**Reply by ilbrigz** [reply]
![image](https://github.com/caprover/caprover/assets/32309779/bd8189fd-9805-4b19-9646-2fa67e7aed61)

Hi again @githubsaturn. I may need some help here. Namecheap actually given me back the domain.

I'm playing around again with caprover and I noticed that I keep getting that warning above in both chrome and firefox. That's probably the reason why my site got detected as phishing/fraud. It's complaining about the ssl certificate being self-signed. Am I missing something here?

Sorry. I'm very new to deployments, domains and everything in between. Thank you

**Reply by githubsaturn** [reply]
What's the root domain that you used for CapRover?

**Reply by ilbrigz** [reply]
I reinstalled everything and everything seems to work. All goods now. Thank you for this awesome project.

**Comment by maietta** [comment]
@ilbrigz, I hope it's a fluke, because I just migrated more than 30 of my domains to them after the mishap at Epik and the remainder of my domains are with Google (I think more than 20), not including client domains. Google Domains just got sold off to Squarespace and I really don't like that, so I have to move my domains elsewhere.

What a mess. I'm sorry to hear that Namecheap disabled your domain(s). That's beyond messed up.
